deploy/helm/events-collector-config.yaml

```diff
@@ -30,7 +30,7 @@ exporters:
       insecure: ${OTEL_ENVOY_ADDRESS_TLS_INSECURE}
     headers:
       "Authorization": "Bearer ${SOLARWINDS_API_TOKEN}"
-      "swi-reporter": "k8s swo-k8s-collector/{{ .Chart.Version }}"
+      "swi-reporter": "swo-k8s-collector/{{ .Chart.Version }} k8s-monitoring"
```

Comment:
Shouldn't the correct format be `k8s-monitoring swo-k8s-collector/{{ .Chart.Version }}` ?